sbom-generator/scan_backoffAlgo.py

```diff
@@ -0,0 +1,55 @@
+import yaml
+from yaml.loader import SafeLoader
+import io
+import os
+from argparse import ArgumentParser
+from sbom_utils import *
+
+spdx_version = 'SPDX-2.2'
+data_license = 'CC0-1.0'
+sbom_creator = 'Amazon Web Services'
+sbom_namespace = 'https://github.com/FreeRTOS/backoffAlgorithm/blob/main/sbom.spdx'
+url = 'https://github.com/FreeRTOS/scan_backoffAlgorithm'
+REPO_PATH = ''
+
+def scan_backoffAlgo():
+    manifest_path = os.path.join(REPO_PATH, 'manifest.yml')
+
+    o = io.StringIO()
```

Comment:
I typically don't like single letter variable names as they are bad for readability.